Src/Plugins/Security/SEC101_053.GoogleGCMServiceAccountValidator.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Utilities;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    public class GoogleGCMServiceAccountValidator : ValidatorBase
```

Comment:
>GoogleGCMServiceAccountValidator [](start = 17, length = 32)

filename should be 015

Reply:
Done

---
In reply to: [586810569](https://github.com/microsoft/sarif-pattern-matcher/pull/256#discussion_r586810569) [](ancestors = 586810569)